plugins/scaffolder-backend-module-gitlab/src/actions/gitlabProjectMigrate.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createRootLogger } from '@backstage/backend-common';
+import { ConfigReader } from '@backstage/config';
+import { ScmIntegrations } from '@backstage/integration';
+import { TemplateAction } from '@backstage/plugin-scaffolder-node';
+import { createGitlabProjectMigrateAction } from './gitlabProjectMigrate';
+import { createMockActionContext } from '@backstage/plugin-scaffolder-node-test-utils';
+import { createMockDirectory } from '@backstage/backend-test-utils';
+
+// Make sure root logger is initialized ahead of FS mock
+createRootLogger();
```

Comment:
Why do we need to do this? Ideally we should drop all dependencies on `backend-common` now as the package is deprecated.

Reply:
Hi @benjdlambert !
I didn't look so much into it, but I found the same call other 6 times, where `import { createMockDirectory } from '@backstage/backend-test-utils'` was used. I built the tests to look similar to others.

If you know a quick hint on how to replace this, I'd appreciate that 😄

Reply:
Hi @benjdlambert 

What should I do with this? Could you give me hand?

Thanks!

Reply:
I think you can just remove it right from these tests, they should also be removed from the other ones, but it's not a prio from this PR. Do the tests fail if this is not called?

Reply:
Seems that it works without 😄 Let me know if you want more changes 🚀